src/client/Client.cc

```diff
@@ -8919,15 +8919,15 @@ loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
     break;
 
   case SEEK_DATA:
-    if (offset < 0 || offset >= in->size) {
+    if (offset < 0 || (uint64_t)offset >= in->size) {
```

Comment:
use `static_cast<>` instead if you please.

Reply:
Sure Kefu. Let me change it accordingly.